--- agg2/font_freetype/agg_font_freetype.cpp.orig	2024-03-06 10:16:58
+++ agg2/font_freetype/agg_font_freetype.cpp	2024-09-10 21:13:37
@@ -171,7 +171,7 @@
             v_control = v_start;
 
             point = outline.points + first;
-            tags  = outline.tags  + first;
+            tags  = (char *)outline.tags  + first;
             tag   = FT_CURVE_TAG(tags[0]);
 
             // A contour cannot start with a cubic control point!
